src/App.css

```diff
@@ -26,12 +26,20 @@ body,
 	align-items: center;
 }
 
-.descope-container {
+.descope-base-container {
 	border-radius: 15px;
 	box-shadow: 0px 1px 50px 0px #b2b2b2;
 	max-width: 400px;
```

Comment:
if for some reason, someone will write `descope-base-container` after `descope-wide-container` - the base 400px width will take precedence.

 is a bit cryptic to base css properties, hard to debug and to maintain, also extensibility is harder because if you want to have something that is only in the non wide flows we will have to override/unset it 
 
I suggest:
1. put only borader-radius and box shadow in the base container
2. create another class `descope-login-container` with the max-width: 400px
3. `const containerClassName = isWideContainer ? 'descope-wide-container' : 'descope-login-container';